docs/reference/files/src-workload-py-file.rst

```diff
@@ -0,0 +1,18 @@
+.. _src-workload-py-file:
```

Comment:
I'd like @medubelko 's opinion on this - the file isn't strictly a necessary part of the charm, and really everything under `src` is a set of Python modules. I'm not sure whether we want that fully documented or maybe want documentation about the `src` directory overall.

Reply:
I'd say it fits the general pattern already established in this section of the docs. We describe fairly conventional files like `CONTRIBUTING.md` and `pyproject.toml`, so I don't think it's a stretch to include this.

I'll leave some suggestions about the wording that could help contextualise it.